src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -321,6 +324,7 @@ const wchar_t* mainWindowTitle = L"Endless Installer";
 
 #define ALL_FILES					L"*.*"
 
+#define CODING_INSTALLER_NAME    L"coding-installer.exe"
```

Comment:
You could check for the presence/absence of `-code-` in the filename, perhaps. And maybe adapt the uninstall path to just check for the presence of `-uninstaller`. This would allow all four states of {installer, uninstaller} × {normal, code}